scripts/build/package.js

```diff
@@ -49,18 +49,19 @@ const licenseToObject = (license) => {
   }
   return license;
 };
-const iconToObject = (icon) => {
+function iconToObject(icon) {
```

Comment:
Arrow function? :thinking: 

```js
const iconToObject = (icon) => {
```

(Don't forget to add the semicolon back at the end of the declaration.)